### PR TITLE
Working

### PR DESCRIPTION
**Pull request by jbants**
Before you submit a pull request, please fill in the following:

**Related Issue(s):**
None

**Description:**
This PR adds the ability to create STAC collections for NALCMS data

**PR checklist:**

- [x] Code is formatted (run `scripts/format`).
- [x] Code lints properly (run `scripts/lint`).
- [x] Tests pass (run `scripts/test`).
- [x] Documentation has been updated to reflect changes, if applicable.
- [x] Changes are added to the [CHANGELOG](../CHANGELOG.md).

### CODE REVIEW

File: examples/collection.json [hunk 1]
```diff
@@ -0,0 +1,191 @@
+{
+  "type": "Collection",
+  "id": "nalcms",
+  "stac_version": "1.0.0",
+  "description": "Land Cover change maps shows class transitions from 2010 to 2015 over Canada, Alaska, the conterminous United States, and Mexico. Changes have been assessed from Land Cover maps at 30 meters resolution.",
```
Comment (jbants) [inline examples/collection.json:5]: Also MODIS at 250m

File: examples/collection.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    {
+      "name": "Natural Resources Canada / Canada Centre Mapping and Earth Observation",
+      "roles": [
+        "producer",
+        "processor"
+      ],
+      "url": "https://www.nrcan.gc.ca"
+    },
+    {
+      "name": "Instituto Nacional de Estad\u00edstica y Geograf\u00eda",
```
Comment (jbants) [inline examples/collection.json:103]: Should we encode unicode?

File: examples/collection.json [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+      "roles": [
+        "producer",
+        "processor"
+      ],
+      "url": "https://www.inegi.org.mx/"
+    },
+    {
+      "name": "Comisi\u00f3n Nacional Forestal",
```
Comment (jbants) [inline examples/collection.json:111]: unicode

File: examples/collection.json (hunk 2 above)
Comment (jbants) [reply examples/collection.json:103]: adding `.encode("utf-8")` to the string doesn't change the output

**Comment by cuttlefish** [comment]
The changes in `scripts/` and `docker/` will likely get overwritten when the files are updated from the template repo.
Are those changes we want to push to the template repo?

**Comment by jbants** [comment]
The formatting changes?
My linter and formatter were arguing.
I'll make an issue about it in templates

--
James

**Pull request merged**